src/app/core/common-components/entity-form/form-field/form-field.component.html

```diff
@@ -0,0 +1,37 @@
+<div
+  *ngIf="
+    _field && mode === 'edit' && _field.editComponent;
+    else displayComponent
+  "
+>
+  <div [class.field-with-tooltip-suffix]="_field.description">
+    <ng-container
+      [appDynamicComponent]="{
+        component: _field.editComponent,
+        config: {
+          formFieldConfig: _field,
+          propertySchema: entity.getSchema().get(_field.id),
```

Comment:
As we are passing the entity as well we can remove this here and only access it in the `EditComponent`.

Reply:
:+1: good point. In fact we don't need it in EditComponent anymore at all, because it's just used to (again ... :roll_eyes: ) re-apply some fallback configs from schema to the field config